camayoc/tests/qpc/utils.py

```diff
@@ -45,7 +45,7 @@ def assert_source_update_fails(original_data, source):
     source.client = orig_client
 
 
-def assert_source_create_fails(source):
+def assert_source_create_fails(source, type=''):
```

Comment:
to avoid overriding the builtin `type` you can rename type to `source_type` for example.